### PR TITLE
250-350px screen navigation bar fix

### PR DESCRIPTION
**Pull request by Aceeri**
Solves #13.

Fixed look

### CODE REVIEW

File: src/_layouts/default.tpl [hunk 1]
```diff
@@ -6,14 +6,14 @@
     	<meta name="viewport" content="width=device-width, initial-scale=1">
 
       <title>Amethyst | Data-oriented and Data-driven Game Engine</title>
-      <link rel="icon" type="image/x-icon" href="/assets/amethyst.ico" />
+      <link rel="icon" type="image/x-icon" href="./assets/amethyst.ico" />
```
Comment (LucioFranco) [inline src/_layouts/default.tpl]: change this to `/`

File: src/_layouts/default.tpl [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
 
 	<!-- Latest compiled and minified CSS -->
      	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
 
       	<!-- Optional theme -->
      	<!--<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css" integrity="sha384-fLW2N01lMqjakBkx3l/M9EahuwpSfeNvV63J5ezn3uZzapT0u7EYsXMjQV+0En5r" crossorigin="anonymous">-->
-      <link rel="stylesheet" href="/css/main.css" />
+      <link rel="stylesheet" href="./css/main.css" />
```
Comment (LucioFranco) [inline src/_layouts/default.tpl]: change this to `/`

**Comment by LucioFranco** [comment]
Once you fix those two. I think this can be merged.

**Comment by LucioFranco** [comment]
If this works on normal sized desktop browser sizes we can merge this!

**Comment by Aceeri** [comment]
It should, this only modifies the navigation if the screen is less than 25em or the combination of all the buttons widths.

**Comment by LucioFranco** [comment]
Okay, @ebkalderon this should be merged!

**Comment by ebkalderon** [comment]
Done!

**Pull request merged**